SConstruct

```diff
@@ -123,7 +123,8 @@ opts.Add('p', 'Platform (same as platform=).', "")
 opts.Add('tools', 'Build Tools (Including Editor): (yes/no)', 'yes')
 opts.Add('gdscript', 'Build GDSCript support: (yes/no)', 'yes')
 opts.Add('libogg', 'Ogg library for ogg container support (system/builtin)', 'builtin')
-opts.Add('libvorbis', 'Ogg Vorbis library for vorbis support (system/builtin)', 'builtin')
+opts.Add('libvorbis',
+         'Ogg Vorbis library for vorbis support (system/builtin)', 'builtin')
```

Comment:
The choices `autopep8` made here are a bit unhappy. I plan to rework all arguments anyway so I'll fix it then.